package.json

```diff
@@ -247,6 +247,7 @@
     "@mui/icons-material": "^5.8.4",
     "@mui/lab": "^5.0.0-alpha.80",
     "@mui/material": "^5.6.4",
+    "atomic-queue": "^5.0.4",
```

Comment:
I used waitQueue elsewhere, can we make these the same? I think they have the same purpose. See obsRecorder.ts.

Reply:
No, this is a different queue type. waitQueue is a simple queue that you're processing yourself for the assertment of OBS signals; atomic-queue is a queue that starts workers to process data written to the queue.

Reply:
They are both queues, yes, but they work differently and for different purposes.